.circleci/scripts/tag-rc-if-necessary.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+PREVRC=$(git for-each-ref --sort=-creatordate --format="%(refname:short)|%(creatordate:unix)" refs/tags/* | grep "0-rc|" | head -n 1)
+
+taggedDate=$(echo $PREVRC | cut -d"|" -f2)
+threeWeekDate=$(date --date "20 days ago" +"%s")
+
+if [[ ${taggedDate} -lt ${threeWeekDate} ]]; then
+  OLDTAGNAME=$(echo $PREVRC | cut -d"|" -f1)
+  NEWTAGNAME=$(echo $OLDTAGNAME | awk -F. '{print $1 "." $2+1 ".0-rc"}')
```

Comment:
@adentong I guess the assumption here is that we are only doing minor releases for M1 from here on out.  If there was a major release, then I guess, it must be done manually, right?

Reply:
Yup that's right.

Reply:
@adentong  I assume that you mean the first thought is right.  And what of the second?

Reply:
Nevermind.  I now understand that this is predicated off of there being a tagged release candidate.

LGTM

Reply:
If we check the config file, then script will be M1 specific. I'd really want to have the same for M1/M2/WooC.